test/model.test.js

```diff
@@ -634,17 +634,19 @@ describe.onServer('Remote Methods', function() {
         // this test was seeing this method (with all aliases) as public
         // 'destroyAll', 'deleteAll', 'remove',
         'create',
-        'upsert', 'updateOrCreate',
+        'upsert', 'updateOrCreate', 'patchOrCreate',
         'exists',
         'findById',
+        'replaceById',
+        'replaceOrCreate',
         'find',
         'findOne',
         'updateAll', 'update',
         'deleteById',
         'destroyById',
         'removeById',
         'count',
-        'prototype.updateAttributes',
+        'prototype.patchAttributes', 'updateAttributes',
```

Comment:
I think we either need to fix [L627](https://github.com/strongloop/loopback/pull/2316/files#diff-e43576aabfd6aa741af8ac58f1b4a699R627) to prepend `prototype.` prefix to `isStatic:false` methods, or change remoting definition of `prototype.patchAttributes` to specify method alias as `prototype.updateAttributes`. This depends on how strong-remoting treats aliases of prototype methods, which is something I am not sure about. Could you please check? It is also worth checking how loopback-connector-remote handles these aliases, to ensure code calling `myModelInstance.updateAttributes` keeps working (assuming the model is attached to a remote-backed datasource).